lib/connection/process.coffee

```diff
@@ -11,28 +32,51 @@ module.exports =
       process.env.HOME || process.env.USERPROFILE
 
   jlpath: -> atom.config.get("julia-client.juliaPath")
-  # TODO: this is very naïve.
-  jlargs: -> atom.config.get("julia-client.juliaArguments").split ' '
+
+  checkExe: (path, cb) ->
+    which = if process.platform is 'win32' then 'where' else 'which'
```

Comment:
@one-more-minute: This doesn't work on Windows when `path` is an actual path. I think we would need to call `where /r path` instead, but that doesn't work if `path` is only `julia`...

Once again, Windows is annoying. We should probably check for success of `spawn("$path", ["-h")` or something like that.